advanced/Scripts/piholeCheckout.sh

```diff
@@ -46,6 +46,10 @@ checkout() {
     local corebranches
     local webbranches
 
+    local binary
+    get_binary_name
+    binary=$(</etc/pihole/ftlbinary)
```

Comment:
How does this work if the file is not in that directory anymore and is just available during binary download?